shoop_tests/campaigns/test_basket_campaigns.py

```diff
@@ -308,3 +309,89 @@ def test_coupon_uniqueness(rf):
     with pytest.raises(IntegrityError):
         second_campaign.coupon = coupon
         second_campaign.save()
+
+
+@pytest.mark.django_db
+def test_basket_free_product(rf):
```

Comment:
Can you separate these tests to own file. This test_basket_campaigns is getting huge.